### DISCUSSION TITLE
Google Translate toolbar showing inside safe exam browser

### DISCUSSION THREAD

**Discussion by jwallacesilverback** [SafeExamBrowser/seb-win-refactoring#820] [Q&A] [answered]
When have customer in the middle east the is taking English assessments do not want any translation.  The google translate toolbar is showing up.  I cannot replicate it the issue so that I can turn it off.  
![googletranslate](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/60243851/d23862c0-15e4-45b4-ab19-cb12f36da625)
This is running on windows 10, latest Chrome, SEB 3.6.

My research says that in SEB the CEF does not use chrome profiles there for does not load plugins.  SEB settings has enable plugins settings but does not seem to load plugins. The translate functionality of chrome seems to be more built in than a plugin.  

Looking for ideas on where to look.

**Comment by danschlet** [comment, marked as answer]
Enable plug-ins is only a setting in SEB for macOS, not Windows.

Are you sure that Google Translate bar is not part of the web page/web application you're using?

**Comment by jwallacesilverback** [comment]
You are so correct.  I did not want it to be our code but it is.  We are leaving the google_translate_element in page and not displaying it.  we now are removing the element.  That seems to have fixed it.

Thanks
